Test/git-issues/git-issue-356.dfy

```diff
@@ -0,0 +1,155 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
```

Comment:
Aha, nice! We should do this in more compiler tests.

Reply:
+1 - this could save a lot of execution time in the tests. In my WIP refactoring to address #632 I've made the default behaviour to execute in every target language, and I can make the equivalent change to do verification separately like this.